spec/system/user_uses_the_editor_spec.rb

```diff
@@ -98,11 +98,14 @@ def fill_markdown_with(content)
   describe "using v2 editor", js: true do
     it "fill out form with rich content and click publish" do
       visit "/new"
-      fill_in "article-form-title", with: "This is a test"
+      fill_in "article-form-title", with: "This is a <span> test"
```

Comment:
Do we need a similar test for the v1 editor or is this not an issue there?

Reply:
It's regardless of the editor version. The bug is actually in the rendering of the article.